File: ElaborateAndEvaluation/PHP/number1.php
<?php

function findNthTerm($firstTerm, $difference, $n) 
{
    return $firstTerm + ($n - 1) * $difference;
}

// part a
$firstTermA = 3;
$commonDifferenceA = 6;
$termA = findNthTerm($firstTermA, $commonDifferenceA, 35);
echo "Term 35 is: " . $termA . PHP_EOL;

// part b
$firstTermB = 4;
$commonDifferenceB = -5;
$termB = findNthTerm($firstTermB, $commonDifferenceB, 125);
echo "Term 125 is: " . $termB . PHP_EOL;

// part c
$term5C = -8;
$term25C = 72;
$n1 = 5;
$n2 = 25;
$commonDifferenceC = ($term25C - $term5C) / ($n2 - $n1);
$firstTermC = $term5C - ($n1 - 1) * $commonDifferenceC;
$termC = findNthTerm($firstTermC, $commonDifferenceC, 100);
echo "Term 100 is: " . $termC . PHP_EOL;

// part d
$term5D = -32;
$term18D = 85;
$n1 = 5;
$n2 = 18;
$commonDifferenceD = ($term18D - $term5D) / ($n2 - $n1);
$firstTermD = $term5D - ($n1 - 1) * $commonDifferenceD;
$termD = findNthTerm($firstTermD, $commonDifferenceD, 36);
echo "Term 36 is: " . $termD . PHP_EOL;

?>

File: ElaborateAndEvaluation/number1.cpp
#include <iostream>

int findNthTerm(int firstTerm, int difference, int n) 
{
    return firstTerm + (n - 1) * difference;
}

int main() 
{
    // part a
    int firstTermA = 3;
    int commonDifferenceA = 6;
    int termA = findNthTerm(firstTermA, commonDifferenceA, 35);
    std::cout << "Term 35 is: " << termA << std::endl;

    // part b
    int firstTermB = 4;
    int commonDifferenceB = -5;
    int termB = findNthTerm(firstTermB, commonDifferenceB, 125);
    std::cout << "Term 125 is: " << termB << std::endl;

    // part c
    int term5C = -8;
    int term25C = 72;
    int n1 = 5;
    int n2 = 25;
    int commonDifferenceC = (term25C - term5C) / (n2 - n1);
    int firstTermC = term5C - (n1 - 1) * commonDifferenceC;
    int termC = findNthTerm(firstTermC, commonDifferenceC, 100);
    std::cout << "Term 100 is: " << termC << std::endl;

    // part d
    int term5D = -32;
    int term18D = 85;
    n1 = 5;
    n2 = 18;
    int commonDifferenceD = (term18D - term5D) / (n2 - n1);
    int firstTermD = term5D - (n1 - 1) * commonDifferenceD;
    int termD = findNthTerm(firstTermD, commonDifferenceD, 36);
    std::cout << "Term 36 is: " << termD << std::endl;

    return 0;
}
